app/sidekiq/handle_test_post_job.rb

```diff
@@ -0,0 +1,7 @@
+class HandleTestPostJob
+  include Sidekiq::Job
```

Comment:
And move it to jobs folder

```suggestion
  include Sidekiq::Workers
```